src/Tizen.NUI.Components/Controls/Switch.cs

```diff
@@ -361,33 +359,5 @@ public class SelectEventArgs : EventArgs
             /// <since_tizen> 6 </since_tizen>
             public bool IsSelected;
         }
-
-        /// <summary>
-        /// Get current track part to the attached SwitchExtension.
-        /// </summary>
-        /// <remarks>
-        /// It returns null if the passed extension is invaild.
-        /// </remarks>
-        /// <param name="extension">The extension instance that is currently attached to this Switch.</param>
-        /// <return>The switch's track part.</return>
-        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
looks like this is removed. is it OK? ^^;

Reply:
Since NUI opened some APIs that enabled access to sub-components to a component,
now Switch has API : Thumb, Track.
So the legacy API `GetCurrentTrack` and `GetCurrentThumb` are not needed anymore.

Reply:
OK. I see. Thank you for explanation~